foodhub_custom/models/foodhub_product.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class FoodhubProduct(models.Model):
+    _inherit = "product.template"
+    location = fields.Char("Location")
```

Comment:
I like to use named parameters for all fields parameters. Here, for example, I would write :

```py
location = fields.Char(string="Location")
# or
location = fields.Char()
```

Whats does @remytms and @huguesdk think? What are your tastes ?

Reply:
OK, for now I used the "string=Location" form.